main.tf

```diff
@@ -14,25 +15,26 @@ data "aws_iam_policy_document" "repo" {
       "ecr:GetAuthorizationToken",
       "ecr:GetDownloadUrlForLayer",
       "ecr:GetRepositoryPolicy",
-      "ecr:ListImages"
+      "ecr:ListImages",
     ]
     principals {
       identifiers = [
```

Comment:
use flatten function to make it as a single list. or TF12 errors out.

Reply:
I asked for the identifiers, as the list contains a list inside called var.cross_account_users.
Please remove for the actions and add it to identifiers.